src/Native/Runtime/CachedInterfaceDispatch.cpp

```diff
@@ -86,7 +86,7 @@ static void * UpdatePointerPairAtomically(void * pPairLocation,
                                           void * pSecondPointer,
                                           bool fFailOnNonNull)
 {
-#if defined(_X86_) || defined(_ARM_)
+#if !defined BIT64
```

Comment:
The unwritten convention is to write this as: `#ifndef BIT64` or `#if !defined(BIT64)`

Reply:
Added parens to match the rest of this file.

Reply:
Changed to positive and used parens